eng/pipelines/scripts/Get-Linting-Commands.ps1

```diff
@@ -49,7 +48,7 @@ Write-Host "Source branch: ${SourceBranch}"
 Write-Host "Target branch: ${TargetBranch}"
 Write-Host "Linting pipeline variable: ${LintingPipelineVariable}"
 
-if ($BuildReason -eq "Scheduled") {
+if ($BuildReason -ne "PullRequest" -and $BuildReason -ne "IndividualCI") {
```

Comment:
Why have IndivudualCI here? I assume you basically want everything except pull request.

Reply:
Maybe I misinterpreted what IndividualCI is, what is the Build.Reason for the post merge pipeline that happens?

Reply:
There are 2 CI reasons IndividualCI and BatchCI however I don't think you will have the correct target branch set in either of those cases so I think you will still want to run across the entire repo instead of against a specific set of files.  I'd suggest you keep this only to PullRequest unless you test the CI scenario.